<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schedule</class>
 <widget class="QWidget" name="Schedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="startGroup">
        <property name="toolTip">
         <string>Set data collection starting conditions.</string>
        </property>
        <property name="title">
         <string>Start</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QRadioButton" name="StartNow">
           <property name="toolTip">
            <string>Start data collection as soon as possible.</string>
           </property>
           <property name="text">
            <string>Immediately</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">startButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QRadioButton" name="StartOn">
             <property name="toolTip">
              <string>Start data collection at a specific time.</string>
             </property>
             <property name="text">
              <string>On   </string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">startButtons</string>
             </attribute>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="StartDateTime">
           <property name="toolTip">
            <string>Programmed date/time to start data collection (UTC)</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="runGroup">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Set data collection ending conditions.</string>
        </property>
        <property name="title">
         <string>Run</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QRadioButton" name="RunIndefinitely">
           <property name="toolTip">
            <string>Collect data for as long as possible, until either battery or storage runs out.</string>
           </property>
           <property name="text">
            <string>Indefinitely</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">runButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="endDateTimePicker">
           <item>
            <widget class="QRadioButton" name="RunUntil">
             <property name="toolTip">
              <string>Stop collecting data at a specific time.</string>
             </property>
             <property name="text">
              <string>Until</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">runButtons</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="EndDateTime">
             <property name="toolTip">
              <string>Programmed date/time to halt data collection (UTC)</string>
             </property>
             <property name="displayFormat">
              <string>M/d/yy h:00 AP</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::UTC</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="periodGroup">
        <property name="toolTip">
         <string>Set Data Sample Period</string>
        </property>
        <property name="title">
         <string>Period</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="period">
           <property name="toolTip">
            <string>Data sample period</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Seconds</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="hibernationGroup">
        <property name="title">
         <string>Hibernation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="startButtons"/>
  <buttongroup name="runButtons"/>
 </buttongroups>
</ui>
